pkg/metrics/test_helper.go

```diff
@@ -70,8 +71,23 @@ func AssertSerieEqual(t *testing.T, expected, actual *Serie) {
 	AssertPointsEqual(t, expected.Points, actual.Points)
 }
 
+type sketchComparator func(exp, act *quantile.Sketch) bool
+
 // AssertSketchSeriesEqual checks whether two SketchSeries are equal
 func AssertSketchSeriesEqual(t assert.TestingT, exp, act SketchSeries) {
+	assertSketchSeriesEqualWithComparator(t, exp, act, func(exp, act *quantile.Sketch) bool {
+		return exp.Equals(act)
+	})
+}
+
+// AssertSketchSeriesApproxEqual checks whether two SketchSeries are approximately equal
+func AssertSketchSeriesApproxEqual(t assert.TestingT, exp, act SketchSeries) {
+	assertSketchSeriesEqualWithComparator(t, exp, act, func(exp, act *quantile.Sketch) bool {
+		return quantile.SketchesApproxEqual(exp, act, .05)
```

Comment:
how to we determine the `.05` in this case? and if it's hardcoded should it be a parameter of `SketchesApproxEqual`

Reply:
This was experimental based on the actual data (I think the max error was like ~2%). I feel like it's useful to be able to choose this based on your test.

Reply:
Oh, I thought I had pushed it up. I can do that at least.